src/Proto.Actor/Configuration/ActorSystemConfig.cs

```diff
@@ -47,6 +48,15 @@ public record ActorSystemConfig
     ///     All props are translated via this function
     /// </summary>
     public Func<Props, Props> ConfigureProps { get; init; } = props => props;
+    
+    /// <summary>
+    ///     Allows ActorSystem-wide augmentation of system Props
+    ///     All system props are translated via this function
+    /// </summary>
+    public Func<string, Props, Props> ConfigureSystemProps { get; init; } = (_,props) => props
+        .WithDeadlineDecorator(TimeSpan.FromSeconds(1))
+        .WithLoggingContextDecorator()
```

Comment:
If we should use a logging decorator by default, should we reduce the log level for messages to trace?
This could quickly become very noisy

Reply:
Done, I've set the loglevel to None. but using Information for system/infra messages, and Error for anything that throws